pylint/checkers/format.py

```diff
@@ -251,13 +252,13 @@ class FormatChecker(BaseTokenChecker, BaseRawFileChecker):
         ),
     )
 
-    def __init__(self, linter=None):
+    def __init__(self, linter: PyLinter):
         super().__init__(linter)
-        self._lines = None
-        self._visited_lines = None
-        self._bracket_stack = [None]
+        self._lines: dict[int, str] = {}
+        self._visited_lines: dict[str | None, int | None] = {}
```

Comment:
This seems incorrect? We use `.get()` with `ints` on this dictionary. Shouldn't it be `dict[int, int]`?